modules/components/AsyncConnect.js

```diff
@@ -56,7 +56,25 @@ export default class AsyncConnect extends Component {
   }
 
   isLoaded() {
-    return this.context.store.getState().reduxAsyncConnect.loaded;
+    const componentKeys = this.props.components.reduce((componentKeysMemo, component) => {
+      if (component && component.reduxAsyncConnect && component.reduxAsyncConnect.length) {
```

Comment:
we could filter components before via `filterAndFlattenComponents(this.props.components)`